spec/visual/samples/racket

```diff
@@ -2,11 +2,20 @@
 
 ;; Single-line comment
 
+;; Multi-line comments
+
 #| Multi-line comment on one line |#
 
 #| Multi-line comment on
    two lines |#
 
+#|
+#  Block comment.
+#  (define x 3)
+#  is not highlighted as the commented text.
```

Comment:
Why `#` on these lines? I believe block comment should be identified just by starting #| and ending |#. Having additional `#` for this use case might be confusing.

Reply:
My bad. Didn't notice Vim had stuck those in there. Will remove.